package.json

```diff
@@ -21,6 +21,10 @@
       "react-native-lab/react-native"
     ]
   },
+  "resolutions": {
```

Comment:
why are resolutions needed?

Reply:
Because `expo eject` (and `prebuild`) change the react-native and expo-updates versions to non-compliant versions which breaks the symlinks and causes new versions of those packages to be installed in `apps/native-component-list/node_modules`. We haven't thought of a good way around this yet.